src/main/scala/firrtl/transforms/DescribeFixed.scala

```diff
@@ -0,0 +1,50 @@
+// See LICENSE for license details.
+
+package firrtl.transforms
+
+import firrtl._
+import firrtl.annotations._
+import firrtl.ir._
+
+/**
+ * Adds description to fixed point nodes giving its width and binary point
+ */
+object DescribeFixed extends Transform {
+  def inputForm = MidForm
+  def outputForm = MidForm
+
+  private def onModule(cir: CircuitName)(m: DefModule): Seq[DescriptionAnnotation] = {
+    val mod = ModuleName(m.name, cir)
+    var descs = Seq[DescriptionAnnotation]()
+    def onType(n: Target)(tpe: Type): Type = {
+      tpe match {
+        case FixedType(IntWidth(w), IntWidth(p)) =>
+          descs = descs :+ DescriptionAnnotation(n, s"Fixed point with binary point $p")
+        case f: FixedType => throw new Exception(s"Uh oh!")
```

Comment:
Better Error message here might be "Found FixedPoint with unresolved width or binary point"

Reply:
err, yes. that's debug stuff i should have caught. thanks!